frontend/src/App.tsx

```diff
@@ -46,24 +46,24 @@ const router = createBrowserRouter([
         path: '/setting',
         element: <Setting />,
       },
-      {
-        path: '/sign-up',
-        element: <SignUp />,
-      },
-      {
-        path: '/login',
-        element: <LoginPage />,
-      },
-      {
-        path: '/forgot-password',
-        element: <ForgotPassword />,
-      },
-      {
-        path: '/reset-password',
-        element: <ResetPassword />,
-      },
     ],
   },
+  {
+    path: '/sign-up',
+    element: <SignUp />,
+  },
+  {
+    path: '/login',
+    element: <LoginPage />,
+  },
+  {
+    path: '/forgot-password',
+    element: <ForgotPassword />,
+  },
+  {
+    path: '/reset-password',
+    element: <ResetPassword />,
+  },
```

Comment:
I think after this PR you should make a PR where the formatter + linter is run on the whole codebase. Reviewing PRs where actual changes are mixed in with formatting changes makes it harder to review.